### PR TITLE
Include spinner in search

### PR DESCRIPTION
**Pull request by exacs**

### CODE REVIEW

File: @kth/style/scss/components/search.scss [hunk 1]
```diff
@@ -44,10 +44,23 @@ $secondary-border-width: math.div(1rem, 16);
...
+    }
+
+    > button.loading::after {
+      @include icons.icon-spinner;
+      content: "";
+      display: block;
+      width: 1rem;
+      height: 1rem;
+      margin: auto;
+      background: var(--color-secondary);
```
Comment (EllenWiderstrand) [inline @kth/style/scss/components/search.scss:63]: Isn't this already set on row 49?

File: @kth/style/scss/components/search.scss [hunk 2]
```diff
@@ -44,10 +44,23 @@ $secondary-border-width: math.div(1rem, 16);
...
+
+    > button:not(.loading)::after {
+      @include icons.icon-search;
+    }
+
+    > button.loading::after {
+      @include icons.icon-spinner;
+      content: "";
+      display: block;
+      width: 1rem;
```
Comment (EllenWiderstrand) [inline @kth/style/scss/components/search.scss:60]: Why not use the same size as for the search icon (1.5rem)?

File: @kth/style/scss/tokens/icons.scss [hunk 3]
```diff
@@ -79,3 +86,19 @@
 @mixin icon-new-window-500 {
   @include with-icon(raw.$icon-new-window-500);
 }
+
+@mixin icon-spinner {
+  // This is based on https://css-loaders.com/spinner/#l3
+  padding: 0.125rem;
+  aspect-ratio: 1;
+  border-radius: 50%;
+  --_m: conic-gradient(transparent 10%, #000),
```
Comment (EllenWiderstrand) [inline @kth/style/scss/tokens/icons.scss:95]: Never seen this notation before so I'm just curios, do you know what it does?

File: apps/style-web/src/pages/style/en/components/search.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (EllenWiderstrand) [inline apps/style-web/src/pages/style/en/components/search.md:1]: Should we include this in `local-navigation.ts` so that it can be viewed?

**Review by EllenWiderstrand — COMMENTED** [review]
Haven't looked at it visually (is it possible to view somewhere?), but left a few questions 😄

File: @kth/style/scss/components/search.scss (hunk 1 above)
Comment (exacs) [reply @kth/style/scss/components/search.scss:63]: Yes! Good catch!

**Comment by exacs** [comment]
@EllenWiderstrand I've decided to change the implementation. For consistency with other icons, I opted for an SVG icon that spins instead.

(I still like the CSS only spinner though :grin:)

**Pull request merged**